challenges/systems/challenge-2.md

```diff
@@ -86,25 +87,25 @@ list of the key observations from the team that affected the result.
 
 ### Server
 
-* Develop a simple authorization scheme that defines what upstreams are
-  available to which clients; this scheme can be statically defined in code
 * Secure communication between client and server using TLS and token
   authentication
```

Comment:
It's not completely obvious what we mean by "token authentication" or how it's supposed to work. Are we just expecting some hard coded tokens in the backend and clients just pass those through to the LB in some way?

Reply:
I assume this meant something like JWT.

Reply:
Updated to say "opaque bearer token".